app/javascript/comments.vue

```diff
@@ -114,10 +114,16 @@ export default {
       return this.$store.getters.checkId
     },
     roleClass() {
-      return `is-${this.currentUser.role}`
+      return `is-${this.currentUser.primary_role}`
     },
     daimyoClass() {
       return { 'is-daimyo': this.currentUser.daimyo }
+    },
+    roleAdmin: function () {
+      return this.currentUser.roles.includes('admin')
+    },
+    roleAdviser: function () {
+      return this.currentUser.roles.includes('adviser')
     }
```

Comment:
関数 `roleAdmin` と `roleAdviser` はどちらも Boolean 型の値( `true` or `false` )を返していますね。
そのような関数には `isHoge` や `hasFuga` のように命名することが多いです。

```suggestion
    isAdmin: function () {
      return this.currentUser.roles.includes('admin')
    },
    isAdviser: function () {
      return this.currentUser.roles.includes('adviser')
    }
```

参考資料を見つけられなかったので、代わりに bootcamp アプリ内のコードを例に示しますと以下のような関数があります

https://github.com/fjordllc/bootcamp/blob/400969fadf45b7baf1a2bf022390afb142f0582e/app/javascript/reports.vue#L75-L77
https://github.com/fjordllc/bootcamp/blob/400969fadf45b7baf1a2bf022390afb142f0582e/app/javascript/answers.vue#L97-L99

こちら、一括で置換して対応できると思いますので、修正をお願いできますでしょうか🙏

Reply:
@AudioStakes 
> 関数 roleAdmin と roleAdviser はどちらも Boolean 型の値( true or false )を返していますね。
そのような関数には isHoge や hasFuga のように命名することが多いです。

おお〜、そうなのですね！ありがとうございます😊いや本当に勉強になります✨
提案いただいた内容で修正しましたので、確認をよろしくお願いいたします🙏